tests/integration/integration_test.go

```diff
@@ -301,6 +303,52 @@ status: {}
 			})
 		})
 	})
+
+	Context("prepare", func() {
+		When("prepare is called without any arguments", func() {
+			It("downloads and installs the latest release of manifest files from the profile repository", func() {
+				cmd := exec.Command(binaryPath, "prepare")
+				output, err := cmd.CombinedOutput()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(string(output)).To(Equal("install finished\n"))
+			})
+		})
```

Comment:
do we want to check that things are actually installed in the cluster?

Reply:
Sure, although to be fair, this will change once there is a wait for ready state implemented in the prepare command.

Reply:
hmmm okay i can wait for that, so long as we eventually test more than a log line 😉

Reply:
Oh, I'm am writing a check, just wanted to know for the future. :D